NOTES.md

```diff
@@ -1,3 +1,191 @@
+# Release 1.16.0 (Stand By Me)
+
+Released on 2015-01-28
+
+The highlights of this release are numerous and exciting. Read the
+[release blog post][1.16-blog] for more details.
+
+[1.16-blog]: http://rethinkdb.com/blog/1.16-release/
+
+There may or may not be API-breaking changes in this release.
+
+## New Features ##
+
+* ReQL admin: a new cluster management and monitoring API (#2169)
+  * Added a system database named `rethinkdb`
+    * The `cluster_config` table contains cluster-wide settings (#2924)
+    * The `current_issues` table lists issues affecting the cluster and suggests solutions (#2864, #3258)
+    * The `db_config` table allows listing and renaming databases (#151, #2871)
+    * The `jobs` table lists running jobs (#3115)
```

Comment:
`running jobs` is rather ambiguous because we've never used that terminology before - I would suggest `running queries and background tasks`.